test/jdk/java/awt/print/PrinterJob/NumCopies.java

```diff
@@ -21,169 +21,62 @@
  * questions.
  */
 
-/**
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+
+import jtreg.SkippedException;
+
+/*
  * @test
  * @bug 4258003
  * @summary Checks the right number of copies are printed
  * @key printer
+ * @library /test/lib /java/awt/regtesthelpers
+ * @build PassFailJFrame jtreg.SkippedException
  * @run main/manual NumCopies
  */
-
-import java.awt.*;
-import java.awt.print.*;
-
 public class NumCopies implements Printable {
 
-
-  public static void main(String[] args) {
-
-  String[] instructions =
-        {
-         "You must have a printer available to perform this test",
-         "This test should print a total of four pages which are two",
-         " copies of each of two pages which consist of the text :-",
-         "'This is page number N', where N is 0 and 1.",
-         "The pages should be uncollated."
-       };
-      Sysout.createDialog( );
-      Sysout.printInstructions( instructions );
-
-    PrinterJob job = PrinterJob.getPrinterJob();
-    job.setCopies(2);
-    job.setPrintable(new NumCopies());
-    try {
+    private static final String INSTRUCTIONS =
+            "You must have a printer available to perform this test\n" +
+            "This test should print a total of four pages which are two\n" +
+            " copies of each of two pages which consist of the text :-\n" +
```

Comment:
This does not look good, I can hardly understand. Could you simplify the sentence/description.

Two pages, two copies of each…